src/oca_github_bot/config.py

```diff
@@ -34,7 +34,7 @@ def func_wrapper(*args, **kwargs):
 GITHUB_SECRET = os.environ.get("GITHUB_SECRET")
 GITHUB_LOGIN = os.environ.get("GITHUB_LOGIN")
 GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN")
-GITHUB_ORG = os.environ.get("GITHUB_ORG")
+GITHUB_ORG = os.environ.get("GITHUB_ORG", "").split(",")
```

Comment:
If GITHUB_ORG is not set, this will return `['']` and then the `for org in GITHUB_ORG` wil not do what we expect.

Reply:
Thanks for your review.
Is it now ok ?

Reply:
Looks good, thanks!